### PR TITLE
[README.md] add WebGL build instructions

### PR DESCRIPTION
**Pull request by msbaines**
## Motivation and Context

Prepare some documentation in time for launch.

## How Has This Been Tested

Verified instructions work.

## Types of changes

- [X] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +231,20 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
...
+
+1. Download the [test scenes](http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip) and extract locally to habitat-sim creating habitat-sim/data.
+1. Download and install [emscripten](https://emscripten.org/docs/getting_started/downloads.html) (version 1.38.38 is verified to work)
+1. Set EMSCRIPTEN in your environment
+   ```bash
+   export EMSCRIPTEN=/pathto/emsdk/fastcomp/emscripten
+1. Build using `./build.js`
+1. Run webserver
+   ```bash
+   python3 -m http.server
```
Comment (apsdehal) [inline README.md]: It is better to specify the host and port directly. Users might have different defaults.

File: README.md [hunk 2]
```diff
@@ -230,6 +231,20 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
 
   We also have a dev slack channel, please follow this [link](https://join.slack.com/t/ai-habitat/shared_invite/enQtNjY1MzM1NDE4MTk2LTZhMzdmYWMwODZlNjg5MjZiZjExOTBjOTg5MmRiZTVhOWQyNzk0OTMyN2E1ZTEzZTNjMWM0MjBkN2VhMjQxMDI) to get added to the channel.
 
+## WebGL
+
+1. Download the [test scenes](http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip) and extract locally to habitat-sim creating habitat-sim/data.
```
Comment (apsdehal) [inline README.md]: Maybe directly provide `wget` based command?

File: README.md [hunk 3]
```diff
@@ -230,6 +231,20 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
...
+1. Download and install [emscripten](https://emscripten.org/docs/getting_started/downloads.html) (version 1.38.38 is verified to work)
```
Comment (apsdehal) [inline README.md]: Same here, these all can be converted to bunch of commands which users can directly copy paste.

File: README.md [hunk 4]
```diff
@@ -230,6 +231,20 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
...
+1. Set EMSCRIPTEN in your environment
+   ```bash
+   export EMSCRIPTEN=/pathto/emsdk/fastcomp/emscripten
+1. Build using `./build.js`
```
Comment (apsdehal) [inline README.md]: I guess this was supposed to be:
```suggestion
1. Build using `./build_js.sh`
```

File: README.md (hunk 1 above)
Comment (msbaines) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (msbaines) [reply README.md]: wget is not available on Mac by default.
I'm also not sure that is easier than downloading a link off the web-site.

File: README.md (hunk 3 above)
Comment (msbaines) [reply README.md]: I'm hesitant to reproduce the emscripten documentation. For Matterport, we just point to their documentation.

File: README.md (hunk 4 above)
Comment (msbaines) [reply README.md]: Fixed. Thanks!

**Pull request merged**